python/lwt_interface/tskit_lwt_interface.h

```diff
@@ -1566,9 +1586,19 @@ write_table_arrays(tsk_table_collection_t *tables, PyObject *dict)
         { "populations", population_cols, tables->populations.metadata_schema,
             tables->populations.metadata_schema_length },
         { "provenances", provenance_cols, NULL, 0 },
+        /* We don't want to insert empty indexes, return an empty dict if there are none
+         */
+        { "indexes",
+            tsk_table_collection_has_index(tables, 0) ? indexes_cols : no_indexes_cols,
+            NULL, 0 },
     };
 
     for (j = 0; j < sizeof(table_descs) / sizeof(*table_descs); j++) {
+        if (strcmp(table_descs[j].name, "indexes") == 0
```

Comment:
Do we need to do this? I think it'd be better if the dict encoding had the empty dictionary as the output rather than excluding the key. This is what the TableCollection does, right?

Reply:
Fixed in https://github.com/tskit-dev/tskit/pull/921/commits/4d815295f13d97eaf2912d04cbdad09e6ef485d2